lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -123,7 +207,13 @@ async def _get_subscribers(self) -> [SubscriberData]:
         SUBSCRIBER_SYNC_LATENCY.observe(
             time_elapsed.total_seconds() * 1000,
         )
-        return subscribers
+
+        return [subscribers, flat_digest]
```

Comment:
Prefer immutability where possible. May also have to update the fn signature
```suggestion
        return (subscribers, flat_digest)
```